gravitee-apim-console-webui/src/management/api/analytics/pathMappings/api-path-mappings.component.ts

```diff
@@ -45,25 +51,36 @@ export class ApiPathMappingsComponent implements OnInit, OnDestroy {
   public displayedColumns = ['path', 'actions'];
   public pathMappingsDS: PathMappingDS[] = [];
   public isLoadingData = true;
+  public isReadOnly = true;
+  private swaggerDocs: Page[];
 
   constructor(
     @Inject(UIRouterStateParams) private readonly ajsStateParams,
     private readonly apiService: ApiService,
     private readonly matDialog: MatDialog,
     private readonly snackBarService: SnackBarService,
+    private documentationService: DocumentationService,
   ) {}
 
   public ngOnInit(): void {
     this.apiService
       .get(this.ajsStateParams.apiId)
       .pipe(
-        tap((api) => {
+        mergeMap((api) => {
           this.api = api;
           this.pathMappingsDS = this.toPathMappingDS(api);
           this.isLoadingData = false;
+
+          this.isReadOnly = api.definition_context.origin === ('kubernetes' as ApiOrigin);
+          return this.documentationService.apiSearch(api.id, {
```

Comment:
Maybe we can do it independently to the `apiservice.get` as we have already access to the api id using this.ajsStateParams.apiId. Wdyt ?